b8/privkey

```diff
@@ -0,0 +1,59 @@
+-----BEGIN PGP PRIVATE KEY BLOCK-----
```

Comment:
inb4 we receive repeated alerts for "accidentally" uploading a private key to github